File: minto/1주차-입출력/1924.js
let input = require('fs').readFileSync('/dev/stdin').toString().split(' ');

let month = +input[0];
let day = +input[1];

let count = 0;

count += day;

for (let i = 1; i < month; i++) {
  switch (i) {
    case 1:
      count += 31;
      break;
    case 2:
      count += 28;
      break;

    case 3:
      count += 31;
      break;

    case 4:
      count += 30;
      break;

    case 5:
      count += 31;
      break;

    case 6:
      count += 30;
      break;

    case 7:
      count += 31;
      break;

    case 8:
      count += 31;
      break;

    case 9:
      count += 30;
      break;

    case 10:
      count += 31;
      break;

    case 11:
      count += 30;
      break;
  }
}

count = count % 7;
switch (count) {
  case 1:
    console.log('MON');
    break;
  case 2:
    console.log('TUE');
    break;
  case 3:
    console.log('WED');
    break;
  case 4:
    console.log('THU');
    break;
  case 5:
    console.log('FRI');
    break;
  case 6:
    console.log('SAT');
    break;
  case 0:
    console.log('SUN');
    break;
}

File: ujin/1주차_입출력/1924.py
month=[31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
week=['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']
a, b = map(int,input().split())
n = a - 1 #1월 1일과의 월 차이
m = b - 1 #1월 1일과의 일 차이
sum = 0
for i in range(0,n):
    sum+=month[i]
sum+=m
result = sum % 7
print(week[result])